### PR TITLE
Add __PHP_Incomplete_Class

### PR DESCRIPTION
**Pull request by haszi**
Add a page on `__PHP_Incomplete_Class`.

### CODE REVIEW

File: reference/igbinary/functions/igbinary-unserialize.xml [hunk 1]
```diff
@@ -63,7 +63,8 @@
        <para>
         It is possible to set a callback function which will be called,
         if an undefined class should be instantiated during unserializing.
-        (to prevent getting an incomplete <type>object</type> <literal>__PHP_Incomplete_Class</literal>.)
+        (to prevent getting an incomplete <type>object</type>
+        <classname>__PHP_Incomplete_Class</classname>.)
```
Comment (Girgias) [inline reference/igbinary/functions/igbinary-unserialize.xml:67]: This reads a bit weird doesn't it?

File: reference/var/functions/unserialize.xml [hunk 2]
```diff
@@ -58,8 +58,9 @@
         <para>
          It's possible to set a callback-function which will be called,
          if an undefined class should be instantiated during unserializing.
-         (to prevent getting an incomplete <type>object</type> "__PHP_Incomplete_Class".)
-         Use your &php.ini;, <function>ini_set</function> or &htaccess; 
+         (to prevent getting an incomplete <type>object</type>
+         <classname>__PHP_Incomplete_Class</classname>.)
```
Comment (Girgias) [inline reference/var/functions/unserialize.xml:62]: Ditto

**Review by Girgias — APPROVED** [review]
Minor nit but LGTM

File: reference/igbinary/functions/igbinary-unserialize.xml (hunk 1 above)
Comment (haszi) [reply reference/igbinary/functions/igbinary-unserialize.xml:67]: Maybe we can fix this paragraph in a follow-up PR?

File: reference/var/functions/unserialize.xml (hunk 2 above)
Comment (haszi) [reply reference/var/functions/unserialize.xml:62]: Yep. This seems to be a copy of the paragraph from `igbinary-unserialize()`. Follow-up PR?

File: reference/igbinary/functions/igbinary-unserialize.xml (hunk 1 above)
Comment (Girgias) [reply reference/igbinary/functions/igbinary-unserialize.xml:67]: Yes

**Pull request merged**